pinot-core/src/main/java/org/apache/pinot/core/operator/combine/GroupByOrderByCombineOperator.java

```diff
@@ -62,24 +63,29 @@
 public class GroupByOrderByCombineOperator extends BaseOperator<IntermediateResultsBlock> {
   private static final Logger LOGGER = LoggerFactory.getLogger(GroupByOrderByCombineOperator.class);
   private static final String OPERATOR_NAME = "GroupByOrderByCombineOperator";
+  public static final int INFINITE_TRIM_THRESHOLD = 1_000_000_000;
```

Comment:
Maybe better to use `MAX_TRIM_THRESHOLD`?

Reply:
done